src/CodeStyle/Core/CodeFixes/Microsoft.CodeAnalysis.CodeStyle.Fixes.csproj

```diff
@@ -16,8 +16,8 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.CodeStyle.Fixes" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes" />
-    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CodeStyle.UnitTests" />
-    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests" />
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CodeStyle.UnitTestUtilities" />
```

Comment:
❔ Why rename this assembly? We can still reference it as a test utilities library even if it's also used for authoring language-independent tests.

Reply:
This causes our Determinism queue on CI to fail - it checks that projects ending with `UnitTests` are not referenced by any other projects that contain unit tests.

Reply:
See https://github.com/dotnet/roslyn/blob/e051790e791e0ca6ed36404db17307bea868306e/src/Tools/BuildBoss/ProjectData.cs#L17-L18

https://github.com/dotnet/roslyn/blob/bc07049760d044969409de0758aa22b3d2a48258/src/Tools/BuildBoss/ProjectCheckerUtil.cs#L214-L230